configure.ac

```diff
@@ -40,7 +40,7 @@ MONO_VERSION_BUILD=`echo $VERSION | cut -d . -f 3`
 # This can be reset to 0 when Mono's version number is bumped
 # since it's part of the corlib version (the prefix '1' in the full
 # version number is to ensure the number isn't treated as octal in C)
-MONO_CORLIB_COUNTER=16
+MONO_CORLIB_COUNTER=17
```

Comment:
It is already 17 in master.
And I have 18 here https://github.com/mono/mono/pull/9503.
So please use 19.
Too bad it isn't an automatic hash of something. :(

Reply:
I think whoever gets to master first, the other will have to rebuild monolite yet again. But we can proceed racing in parallel..

Reply:
I've ceded 18. You can have it. I'm using 20 and 21.
But 17 is in master and stomping it here I think will break things.
Might need to make another PR to rebuild that.